.github/silabs-builds-mg24-trustzone.json

```diff
@@ -0,0 +1,18 @@
+{
+    "standard": {
+        "default": [
+            {
+                "boards": ["brd4187c"],
```

Comment:
Does this mean that Trustzone is not supported/tested on series3  / 917?

Reply:
I have added create separate lighting-app-series2.slcw and lighting-app-series3.slcw, so we should be able to support at least Series2 and Series3.